CHANGES.rst

```diff
@@ -8,6 +8,10 @@ general
 
 - Fix random seed bug in PSF fitting methods [#862]
 
+- Fix regression test ``compare_asdf`` function replacing use of
+  ``asdf.commands.diff`` with ``deepdiff`` and add ``deepdiff` as
```

Comment:
Missing one closing quote after the second ``deepdiff``.

Reply:
Nice eye for detail :) Fixed in: 537210d219515a348bb04bedca6f523bcfb3feed